examples/src/main/java/org/apache/spark/examples/ml/JavaCountVectorizerExample.java

```diff
@@ -54,6 +54,7 @@ public static void main(String[] args) {
       .setOutputCol("feature")
       .setVocabSize(3)
       .setMinDF(2)
+      .setBinary(true)
```

Comment:
I agree with @MLnick on not to set the binary param in the examples, it completely changes the output and it's easy enough for the user to figure out how to set themselves.